umh-core/examples/example-config-dataflow.yaml

```diff
@@ -0,0 +1,68 @@
+# Copyright 2025 UMH Systems GmbH
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+agent:
+  metricsPort: 8080
+
+redpanda:
+  desiredState: stopped
+
+# Data Flow Components
+dataFlowComponents:
+  - name: data-flow-hello-world
+    desiredState: active
+    versionUUID: 123e4567-e89b-12d3-a456-426614174000
```

Comment:
does versionUUID make sense? why not simply use UUID?

Reply:
versionUUID is in here, to already provide information for the upcoming versioning feature.

Reply:
removed, due to our communication today.